app/views/test_executions/_execution_results.html.erb

```diff
@@ -131,4 +131,19 @@
       <% end %>
     </div>
   <% end %>
+
+<% elsif execution.status_with_sibling == 'errored' %>
+  <% [execution, execution.sibling_execution].each do |ex| %>
+    <% ex_type = ex.task._type[0, 2] %>
+    <% if ex.errored? %>
+      <p class="lead row bg-warning execution-status"><i class="fa fa-fw fa-exclamation-circle text-warning" aria-hidden="true"></i> <%= ex_type %> Execution: An internal error occured</p>
+    <% elsif ex.passing? %>
+      <p class="lead row bg-success execution-status"><i class="fa fa-fw fa-check-circle text-success"></i> <%= ex_type %> Execution passed</p>
+    <% elsif ex.failing? %>
+      <p class="lead row bg-danger execution-status">
+        <i class="fa fa-fw fa-times-circle text-danger" aria-hidden="true"></i>
+        <%= execution_failure_message(ex) %>
```

Comment:
Might want to add `ex_type` to the failure message as well as the errored and passing ones.